### ISSUE TITLE
Remove use of types module constants in twisted/words

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8460]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8460|
|Type|enhancement|
|Created|2016-06-12 06:06:51Z|
|Branch|https://github.com/twisted/twisted/tree/8460-rodrigc-words-notypes|

The *types* module constants have been removed in Python 3.
For example using *types.TupleType* in Python 3 will give an *AttributeError*

These constants should be removed and replaced with the primitive type instead.  See:

http://www.diveintopython3.net/porting-code-to-python-3-with-2to3.html#types

<details><summary>Searchable metadata</summary>

```
trac-id__8460 8460
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8460_rodrigc_words_notypes 8460-rodrigc-words-notypes
branch_author__ 
status__closed closed
resolution__fixed fixed
component__words words
keywords__None None
time__1465711611968466 1465711611968466
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>
cc__ralphm
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc commented|
|-|-|

https://github.com/twisted/twisted/pull/379

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 7d194cd0487d625e2806f9d307a0c1bc5d8201bf
```
#!CommitTicketReference repository="" revision="7d194cd0487d625e2806f9d307a0c1bc5d8201bf"
Merge pull request #379 from twisted/8460-rodrigc-words-notypes

Author: rodrigc
Reviewer: markrwilliams
Fixes: #8460

Remove use of types module in twisted/words
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
